### ISSUE TITLE
Is "m_graph.outputDelta" always a power of 2?

### ISSUE THREAD

**Issue by awai54st** [HPCE/hpce-2016-cw6#29]
Hi,
In your tests, is "m_graph.outputDelta" always a power of 2?
Thank you
Best, awai54st

**Comment by m8pple** [comment]
No, its an arbitrary integer - as a comp-eng person i have an unconcious affinity for powers of two, but the physicists who use the simulator are likely to choose a multiple of 10, or a value related to the timescale of the underlying physical process.

However, its worth noting any optimisations you could see in your discussion document. If there is a good enough reason for choosing powers of two, they could probably be convinced.
